challenge-03/qosf-monthly-challenge-03.ipynb

```diff
@@ -0,0 +1,56 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Challenge 03: Minimum Makespan with Quantum Annealing\n",
+    "\n",
+    "Niranjan owns a catering service and receives orders one day in advance. Each of the chefs he employs is payed by the hour. In order to optimize the cost of running his business, every night, he looks at all of the dishes he needs to deliver for the next day and asks his employees to work the specific hours he requires them for.\n",
+    "\n",
+    "Given an order queue of length $N$ with each order $i$ taking $L_i$ time to prepare, if there are $m$ chefs, design a QUBO/Ising Hamiltonian to help Niranjan find the lowest time it would take for the $m$ chefs to prepare all $N$ dishes. \n",
+    "\n",
```

Comment:
I think this is a little bit too abstract for many people.
Perhaps add two or three concrete cases to work on. Like: "Case 1: number of chefs: 3, tasks length: [1,2,2,2,3,4,5]". It will make it more concrete and easier for people to start working with. 
The first case should be trivial and quickly solvable by hand, so that people have a good place to start.

Reply:
Added a few example cases.